hp-ux/generate_wazuh_packages.sh

```diff
@@ -59,6 +63,24 @@ build_environment() {
     /usr/local/bin/depothelper -f regex
     cp /usr/bin/perl /tmp/perl
     cp /usr/local/bin/perl5.10.1 /usr/bin/perl
+
+    # Install GCC 9.4
+    mkdir ${build_tools_path}
+    cd ${build_tools_path}
+    mkdir bootstrap-gcc
+    cd ${build_tools_path}/bootstrap-gcc
+    curl -k -SO https://packages.wazuh.com/utils/gcc/gcc_9.4_HPUX_build.tar.gz
```

Comment:
HTTTPS? Usually it doesn't work in HP-UX

Reply:
It has worked in every instance from SiteOX provided during this development. However, we can set the `http` URL just in case.